lib/puppet/util/windows/daemon.rb

```diff
@@ -1,12 +1,14 @@
 require 'puppet/util/windows'
```

Comment:
can you, in a separate commit, remove the require to `puppet/util/windows`, and instead require & extend/include `puppet/ffi/windows`?